src/main/java/frc/robot/subsystems/swerve/SwerveSS.java

```diff
@@ -45,6 +46,8 @@ public SwerveSS() {
      *                      each module's speed
      */
     public void drive(Translation2d translation, double rotation, boolean fieldRelative, boolean isOpenLoop) {
+        translation = new Translation2d(translation.getY(), -translation.getX());
+        rotation *= -1;
```

Comment:
please write documentation as to why this is. to me it seems this might be solved by changing the motor ids (it thinks the side is the front)

Reply:
no, it's because WPILib works with positive y is to the left, and positive x is forwards, and CCW+.

Reply:
1. why hasn't this been an issue up to now
2. add documentation